### ISSUE TITLE
ES5 version of LazyStream is not included

### ISSUE THREAD

**Issue by j-fuentes** [mozilla-rpweb/react-lazylog#5]
In the [README.md](https://github.com/mozilla-rpweb/react-lazylog/blob/master/README.md#getting-started) it says that ES5 versions of components are available.

In fact, in the example snipped it imports `'react-lazylog/lib/LazyLog.es5'` and `'react-lazylog/lib/LazyStream.es5'`.

I managed to import the first one, but not `LazyStream.es5`.

I have inspected inside `node_modules/react-lazylog/lib` and these are the files:

```
LazyLog.es5.js  LazyLog.es5.js.map  LazyLog.js  LazyLog.js.map
```

I've tried to build the package by myself (by cloning the repo (v2.0.0) and `yarn && yarn build`), I got the same result:

```
→ yarn build
yarn run v1.1.0
$ neutrino build && neutrino build --options.legacy
✔ Building project completed
Hash: 06d4d0a7decb0dc804ce
Version: webpack 3.7.1
Time: 2967ms
         Asset     Size  Chunks             Chunk Names
    LazyLog.js  23.2 kB       0  [emitted]  LazyLog
LazyLog.js.map  59.4 kB       0  [emitted]  LazyLog
✔ Building project completed
Hash: 22689990862c24805dc8
Version: webpack 3.7.1
Time: 2708ms
             Asset     Size  Chunks             Chunk Names
    LazyLog.es5.js  28.3 kB       0  [emitted]  LazyLog
LazyLog.es5.js.map  59.4 kB       0  [emitted]  LazyLog
✨  Done in 10.86s.
```

I guess something is wrong in `.neutrinorc.js`.

Any clue on how to fix it?

Thanks in advance.

**Comment by j-fuentes** [comment]
I think the readme is wrong and `LazyLog.es5.js` contains everything.

**Comment by eliperelman** [comment]
@j-fuentes good catch! Yeah, it appears the README is wrong, and you can import everything from the `LazyLog.es5.js` file. I'll update that.

**Comment by eliperelman** [comment]
Fixed in 654fadb5b535dc8ec0901a6d79f7d3903214533a. Released in v2.0.1.

**Comment by j-fuentes** [comment]
Thank you so much!
